src/trove/string-dict.scrbl

```diff
@@ -225,6 +236,102 @@ check:
 end
 }
 
+@sd-method["merge"
+  #:contract (a-arrow (SD-of "a") (SD-of "a") (SD-of "a"))
+  #:args (list (list "self" #f) (list "other" #f))
+  #:return (SD-of "a")
+]
+
+Returns a new immutable string-dict that has the keys of both
+the original string-dict and the @pyret{other} string-dict. Choose a key's
```

Comment:
I wouldn't use the verb "choose" here, since that sounds to me like the programmer has a choice.  I'd reword this to "In cases where both input dictionaries have the same key, the resulting value comes from the `@pyret{other}` string-dict."